.travis.yml

```diff
@@ -7,15 +7,6 @@ matrix:
     - docker
     env:
     - OS_NAME: linux
```

Comment:
I suppose that if you are dropping `osx` you don't need the build matrix anymore, right?

Reply:
Yeah, good observation! I'm going to simplify it right away!